libs/index.md

```diff
@@ -215,11 +218,13 @@ Available query params:
   // Render live badge examples
   import { html, render } from 'https://cdn.jsdelivr.net/npm/lit-html@0.10.2/lit-html.js'
 
-  const genExamples = (badges) => html`
+  const genExamples = (badges, docs) => html`
     <h4 id="live-badge">Live Badge</h4>
     <div>${Object.entries(badges).map(([service, examples]) => html`
       <dl>
-        <dt id="${service}"><a href="#${service}">${service}</a></dt>
+        <dt id="${service}"><a href="#${service}">${service}</a>
+        ${docs[service] ? html`<a href="docs/${docs[service]}" target="_blank">Docs</a>` : ''}
```

Comment:
I thought about using a unicode char for this, maybe `ⓘ`?

Reply:
I don't know if it's visible enough.. What do you think?

![image](https://user-images.githubusercontent.com/17952318/43677129-230e12ce-97fd-11e8-941f-659309705583.png)

Reply:
And I thought put it right after the "packagephobia" title, maybe a `⌂` too. That's my whole plan 😂

Reply:
Visibility won't be a problem, it's an enhancement other than core feature, once people clicked it, he will get it. I hope it won't distract too much attention.

Anyway, the feature is all good, happy to merge it 🤗
Visual things might change regularly, they are not blocking issue.